xds/internal/balancer/clusterresolver/resource_resolver.go

```diff
@@ -224,6 +224,16 @@ func (rr *resourceResolver) stop() {
 	for _, r := range cm {
 		r.r.stop()
 	}
+
+	// stop() is called when the LB policy is closed or when the underlying
+	// cluster resource is removed by the management server. In the latter case,
+	// an empty config update needs to be pushed to the child policy to ensure
+	// that a picker that fails RPCs is sent up to the channel.
+	select {
+	case <-rr.updateChannel:
+	default:
+	}
```

Comment:
This makes me wonder whether it's possible for this to race with another source?  If not, maybe a comment about this?

Reply:
Thanks @dfawley. Looks like there is a small race possible in the eds resource resolver. I will fix that ping the PR again.

Reply:
Done. PTAL. Thanks.